### DISCUSSION TITLE
EXE for noobs

### DISCUSSION THREAD

**Discussion by LiamorLG** [ParisNeo/lollms-webui#170] [Ideas]
In the last 3 weeks i have tried installing everything i could find to try to get a local agentgpt style agent running locally on an easy ui. I keep failing and tearing my hair out. Here's what I know how to do:

Reset windows to a clean install.
Navigate to a web page
Download exe
run exe
double-click desktop icon
type words

please make this tech available to someone with such a skillset. ie90% of windows users

**Comment by ParisNeo** [comment]
There is already something like that. It is called gpt4all-chat. You can find it here:
https://github.com/nomic-ai/gpt4all-chat

**Comment by thebaldgeek** [comment]
I have spent hours and kWh's training gpt4all and getting it working really well. Very happy with that process and Desktop GUI.

I just needed a web interface for it for remote access.
Google and this Github suggest that lollms would connect to 'localhost:4891/v1'.  (Yes, I have enabled the API server in the GUI)
I have lollms running on localhost:9600 and all I see an offer to import a blank zoo? (And personalities zoos and extension zoos?).

Can someone please tell me where in lollms I configure my all setup and trained GPT4ALL??
I just need a web interface vs the windows GUI one.
Is lollums openAPI compatible or not?
